dopamine/experiment4/experiment4g/experiment4g_analysis.jl

```diff
@@ -0,0 +1,63 @@
+using CeDataAnalysis
+using DataFrames
+using CSV
+
+
+
+# CREATE DATAFRAME OF FILE NUMBERS AND IDS
+files = DataFrame()
+files.num = 1:51
+files.id = [
```

Comment:
Hm I think that is possible yes. I can definitely name the files with the ID. Would that work?

Reply:
Actually, now that I think about it more, naming each individual file seems like it would take even more work than just doing what I do now and defining the range of files with the same ID. Would it be worth it to do that extra work? And code how to pull that info too?